activities/models.py

```diff
@@ -4,37 +4,43 @@
 
 
 class Activity(models.Model):
-    name = models.CharField(max_length=50)
-    description = models.TextField(blank=True, default='n/a')
+    name = models.CharField(max_length=150)
+    description = models.TextField(blank=True)
     hide_activity = models.BooleanField(default=False)
-    hide_participants = models.BooleanField(default=False)
-    cost = models.CharField(max_length=150, default='n/a')
-    location = models.CharField(max_length=150, default='n/a')
-    start_date = models.DateTimeField(null=True,
-                                      blank=True,
-                                      verbose_name='Start date')
-    end_date = models.DateTimeField(null=True,
-                                    blank=True,
-                                    verbose_name='End date')
+    hide_participants = models.BooleanField(default=False,
+                                            help_text="Participants will be hidden for everyone except the organizers.")
+    cost = models.DecimalField(max_digits=8,
+                               decimal_places=2,
+                               help_text="Used for display only, can't be used to automatically debit people "
+                                         "using Q-rekening (as of yet).",
+                               null=True,
+                               blank=True)
```

Comment:
Ik heb toch liever een `default='n/a'` bij cost om te voorkomen dat mensen denken dat het gratis is

Reply:
Het is beter om dat in de view/template te doen, als in: `<span>Cost: {% if not activity.cost %}unspecified{% else %}{{ activity.cost }}{% endif %}</span>`